src/components/buttons/ButtonsContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { View } from 'react-native'
+
+import { styled } from '../hoc/styled'
+import { MainButton, MainButtonType } from '../themed/MainButton'
+
+export interface ButtonContainerProps {
+  label: string
+  onPress: () => void | Promise<void>
+  disabled?: boolean
+}
+
+interface Props {
+  buttons: Map<MainButtonType, ButtonContainerProps>
+  layout: 'row' | 'column'
+}
+
+/**
+ * A consistently styled view for displaying button layouts.
+ */
+export const ButtonsContainer = React.memo(({ buttons, layout = 'column' }: Props) => {
+  const renderButton = (type: MainButtonType, buttonProps?: ButtonContainerProps) => {
+    if (buttonProps == null) return null
+    const { label, onPress, disabled } = buttonProps
+    return <MainButton key={label} label={label} onPress={onPress} type={type} marginRem={0.5} disabled={disabled} />
+  }
+
+  const renderButtons = () => {
+    return layout === 'column' ? (
+      <>
+        {renderButton('primary', buttons.get('primary'))}
+        {renderButton('secondary', buttons.get('secondary'))}
+        {renderButton('escape', buttons.get('escape'))}
+      </>
+    ) : (
+      <>
+        {renderButton('escape', buttons.get('escape'))}
+        {renderButton('secondary', buttons.get('secondary'))}
+        {renderButton('primary', buttons.get('primary'))}
+      </>
+    )
+  }
+
+  return <StyledButtonContainer layout={layout}>{renderButtons()}</StyledButtonContainer>
+})
+
+const StyledButtonContainer = styled(View)<{ layout: 'row' | 'column' }>(props => {
+  const isRow = props.layout === 'row'
+
+  return {
+    flexDirection: isRow ? 'row' : 'column',
+    justifyContent: 'space-between',
+    padding: props.theme.rem(0.5),
+    paddingHorizontal: isRow ? undefined : props.theme.rem(1)
```

Comment:
There is a layout called "row-reverse". You *could* use this instead of flipping the button order in the render function.